byway/src/main/webapp/home.js

```diff
@@ -138,7 +163,7 @@ async function initMap(start, end, route, keyString) {
  * @param {Array} [waypoints] array of waypoint objects
  */
 async function setRoute(directionsService, directionsRenderer, start, end, waypoints){
-  for(let i = 0; i<20; i++){
+  for(let i = 0; i<30; i++){
```

Comment:
You keep bumping this number up; if ~5 tires is not enough, you may need to use a longer delay.

Reply:
fixed